package.json

```diff
@@ -39,7 +43,7 @@
     "eslint-plugin-promise": "^3.5.0",
     "eslint-plugin-react": "^7.5.1",
     "eslint-plugin-standard": "^3.0.1",
-    "react": "^15.4.2",
-    "react-native": "^0.39.0"
+    "react": "*",
```

Comment:
I'm afraid of this wildcard may cause lots of potential issues, `"react": "^15.4.2||^16"` would better.